core/DataAccess/RawLogUpdater.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\DataAccess;
+
+use Piwik\Common;
+use Piwik\Db;
+
+class RawLogUpdater
```

Comment:
I don't have an issue with names as long as they don't contain `Data` or `Info` :) Maybe the first or last one?